packages/@okta/vuepress-theme-prose/assets/css/layouts/_darktheme.scss

```diff
@@ -508,4 +554,152 @@
   .api-uri-put strong {
     color: cv('link');
   }
-}
\ No newline at end of file
+  .signup--form .fa-github:before {
+    color: cv('text', 'black');
+  }
+  ul.menu--items > li.expandable {
+    & > span:after {
+      background-image: url(/img/icons/chevron-down-white.svg) !important;
+    }
+    & > ul.submenu--items {
+      background: cv('gray', '700');
+      .menu--divider {
+        border-color: cv('gray', '100');
+      }
+      &:before {
+        border-bottom-color: cv('gray', '700');
+      }
+      li a {
+        color: cv('gray', '100');
+      }
+      .menu--icons .menu--icon svg path {
+        fill: cv('gray', '100');
+      }
+    }
+  }
+
+  .magic-box-suggestions {
+    overflow: hidden;
+    border-radius: 4px;
+    background-color: transparent !important;
+  }
+
+  .coveo-magicbox-suggestions {
+    border: none !important;
+  }
+  
+  .magic-box-suggestion {
+    background: cv('gray', '700') !important;
+    border-color: cv('gray', '300') !important;
+    span {
+      color: cv('gray', '100') !important;
+      div {
+        color: cv('gray', '100') !important;
+      }
+    }
+  }
+
+
+  #SearchPage {
+    .coveo-facet-value.coveo-selected .coveo-facet-value-checkbox {
+      background-color: cv('gray', '100');
+    }
+    .coveo-facet-value.coveo-selected .coveo-facet-value-checkbox svg .coveo-hook-svg {
```

Comment:
Code duplicate. This should be parent selector:
.coveo-facet-value.coveo-selected .coveo-facet-value-checkbox